### PR TITLE
added task detail controller

### PR DESCRIPTION
**Pull request by ankitgoyalgloballogic**
#REF: #MOBILEAPPS-1302

### CODE REVIEW

File: ContentApp/PresentationLayer/Tasks/Task List/Controller/Task Detail/TaskDetailViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import AlfrescoContent
+
+class TaskDetailViewModel {
```
Comment (gl-lovekesh) [inline ContentApp/PresentationLayer/Tasks/Task List/Controller/Task Detail/TaskDetailViewModel.swift:22]: Create a common view model class then inherit, 
there are many functions are same in two different view model.

File: ContentApp/PresentationLayer/Tasks/Views/Table Cells/Priority Table Cell/PriorityTableCellViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        return "PriorityTableViewCell"
+    }
+    
+    init(title: String?,
+         priority: Int?) {
+        self.title = title
+        self.priority = priority ?? 0
+    }
+    
+    var taskPriority: TaskPriority {
```
Comment (gl-lovekesh) [inline ContentApp/PresentationLayer/Tasks/Views/Table Cells/Priority Table Cell/PriorityTableCellViewModel.swift:36]: Please use this commonly,  put this function in common enum.

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]
Please the changes as per comment and discussion.

File: ContentApp/PresentationLayer/Tasks/Task List/Controller/Task Detail/TaskDetailViewModel.swift (hunk 1 above)
Comment (ankitgoyalgloballogic) [reply ContentApp/PresentationLayer/Tasks/Task List/Controller/Task Detail/TaskDetailViewModel.swift]: Done

File: ContentApp/PresentationLayer/Tasks/Views/Table Cells/Priority Table Cell/PriorityTableCellViewModel.swift (hunk 2 above)
Comment (ankitgoyalgloballogic) [reply ContentApp/PresentationLayer/Tasks/Views/Table Cells/Priority Table Cell/PriorityTableCellViewModel.swift]: Done

**Review by gl-lovekesh — APPROVED** [review]

**Pull request merged**